terraform/modules/hub/files/cloud-init/prometheus.sh

```diff
@@ -131,10 +131,42 @@ Environment=HTTP_PROXY=${egress_proxy_url_with_protocol}
 Environment=HTTPS_PROXY=${egress_proxy_url_with_protocol}
 EOF
 
-cat <<EOF > /etc/prometheus/prometheus.yml
-${prometheus_config}
-EOF
-
 systemctl daemon-reload
 systemctl enable  prometheus
 systemctl restart prometheus
+
+cat <<EOF >/usr/bin/cronitor-prometheus-config-update.sh
+#!/usr/bin/env bash
+set -ueo pipefail
+
+: "${CRONITOR_URL:?CRONITOR_URL not set}"
+
+function cleanup {
+  curl -sf -m 10 $CRONITOR_URL/fail
+}
+
+trap cleanup ERR
+
+curl -sf -m 10 $CRONITOR_URL/run
+
+aws s3 cp \
+  $CONFIG_BUCKET/prometheus/prometheus.yml \
+  /tmp/prometheus.yml \
+&& \
+if !cmp -s /tmp/prometheus.yml /etc/prometheus/prometheus.yml; then \
```

Comment:
You need a space after the `!`. 

Also personally I'd prefer having this written as `[ ! $(...) ]` but I don't think we have a bash style guide so 🤷‍♀️

Reply:
OPINIONS

I would prefer `! [ ` :D

Reply:
Oh god.

Reply:
I fixed it up in Sakis' way because he spoke first. 😛

Reply:
Thanks!